tests/sklearn/test_sklearn_autolog.py

```diff
@@ -668,56 +668,55 @@ def test_autolog_emits_warning_message_when_model_prediction_fails():
         # Count how many times `mock_warning` has been called on not-fitted `predict` failure
         call_count = len([args for args in mock_warning.call_args_list if msg in args[0][0]])
         # If `_is_plotting_supported` returns True (meaning sklearn version is >= 0.22.0),
-        # `mock_warning` should have been called twice, once for metrics, once for artifacts.
-        # Otherwise, only once for metrics.
-        call_count_expected = 2 if mlflow.sklearn.utils._is_plotting_supported() else 1
+        # `mock_warning` should have been called three times: once for generic estimator metrics,
+        # once for the `training_score` metric, and once for artifacts. Otherwise, just twice for
+        # generic estimator metrics and the `training_score` metric
+        call_count_expected = 3 if mlflow.sklearn.utils._is_plotting_supported() else 2
```

Comment:
The extra expected call count crept in when the `_log_posttraining_metadata` method was moved from `mlflow.sklearn` to `mlflow.sklearn.utils`.